docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '3'
+
+services:
+
+  eventbus:
+    image: rabbitmq:alpine
+
+  influx:
+    image: influxdb:alpine
+    # volumes:
```

Comment:
Need to leave these in?

Reply:
If you uncomment it, you get persistent data storage. That seemed like behavior that's occasionally useful for development, but not often enough to add it as a configuration option.